src/components/Todo-List.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="todo-list">
+    <todo-add @todoAdded="addTodo"></todo-add>
     <ul>
       <li v-for="todo in todos" class="todo-item">
```

Comment:
Looks like I'm getting warnings about now key present. Ok for now, but we should probably add this value to the model once we get a back end wired up. https://vuejs.org/v2/guide/list.html#Components-and-v-for